docs/modules/nifi/examples/getting_started/getting_started.sh

```diff
@@ -133,15 +133,15 @@ metadata:
   name: simple-nifi
 spec:
   image:
-    productVersion: 1.18.0
+    productVersion: 1.21.0
     stackableVersion: "0.0.0-dev"
   clusterConfig:
-    listenerClass: external-unstable
     authentication:
       method:
         singleUser:
           adminCredentialsSecret: nifi-admin-credentials-simple
           autoGenerate: true
+    listenerClass: external-unstable
```

Comment:
Whats the intention? Sort alphabetically? ;)

Reply:
Si :)